cmd/sql_exporter/main.go

```diff
@@ -77,8 +80,14 @@ func main() {
 	if *enableReload {
 		http.HandleFunc("/reload", reloadCollectors(exporter))
 	}
-	klog.Infof("Listening on %s", *listenAddress)
-	klog.Fatal(http.ListenAndServe(*listenAddress, nil))
+
+	promlogConfig := &promlog.Config{}
+	logger := promlog.New(promlogConfig)
```

Comment:
I see that `exporter-toolkit/web` is using `go-kit/log` for logging, while this project currently is based upon `klog/v2`. I think we might eventually want to try to migrate it to `go-kit/log`, but it's out of scope for this pull request.

I believe we can follow the path of [Prometheus](https://github.com/LeviHarrison/prometheus/commit/9bb3555fe4cf2f1d93d2f1d665163ec18365af21) and use [klog-gokit](https://github.com/simonpasquier/klog-gokit#usage) package as a bridge between these two.

It's a minor change, so shouldn't take long. Please let me know if you need any assistance, I'm happy to help. 👍

Reply:
I prefered to got for go-kit/log; I had some issues using klog-gokit

Reply:
it complains about the the argument's type of klog.SetLogger method
honestly, I don't know what to do now ... so better if you have some time and check

Reply:
@lucian-vanghele, Sure! I'll take a look and keep you posted. 👍😃

Reply:
@lucian-vanghele I've taken a look, it's up and running. I'll try to push changes to your PR in the coming days.